docs/SIL.rst

```diff
@@ -705,6 +715,20 @@ generic constraints:
   * Runtime-sized types
   * Non-class protocol types
   * @weak types
+  * Types that can't satisfy the requirements for being loadable because they
+    care about the exact location of their value in memory and need to run some
+    user-written code when they are copied or moved. For example:
```

Comment:
Added "Most commonly, types "care" about the addresses of values because addresses of values are registered in some global data structure, or because values may contain pointers into themselves."

I didn't want to just add ", or have a pointer into itself" because I wanted to distinguish the principle from the consequence.